plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableHandle.java

```diff
@@ -185,12 +195,24 @@ public String getTableName()
         return tableName;
     }
 
+    @Override
+    public boolean managed()
+    {
+        return isManaged();
```

Comment:
Just curious: Why not use `isManaged()` in the interface?

Reply:
Either of the classes will be upset (the record, or the POJO with getters).

Reply:
.. and since we try to use records, we eventually need to embrace the inevitable that getter names will less and less be prefixed with `get`

Reply:
The getters that don't start with `get` technically don't need a `@JsonIgnore` but might make it clear to put it in anyway. Up to you.